pages/about-us/donate/inc/_faq.en.text

```diff
@@ -1,6 +1,6 @@
-h2. How do I do a wire/bank transfer?
+h2. Why don't you accept wire transfers in other currencies?
 
-If you want to make an ACH wire transfer, please contact us at treasurer@riseup.net for information on our ABA/SWIFT/IBAN number. *Please note:* We have to pay $13 for every incoming wire transfer and it is often very difficult for people to successfully transfer money to us this way. We would therefore encourage you to use another method.
+We are charged a large fee for converting other currencies. Additionally, sending directly to our US credit union often is difficult and does not work, and when it does work, we get charged a $13 fee for each transfer, so it is not economical.
 
 h2. How do I donate from outside the U.S?
```

Comment:
I notice that we don't have details about how to do a bank transfer/wire for USD. Do we want to include something for that?